tests/resources/telescope_positions-North-utm.ecsv

```diff
@@ -7,17 +7,27 @@
 # - {name: utm_north, unit: m, datatype: float32}
 # - {name: alt, unit: m, datatype: float32, description: telescope position upwards relative to NNN}
 # - {name: geo_code, datatype: string, description: Geographical Position Code}
-# meta: !!omap
-# - data_type: positionfile
-# - array_name: "North-ArrayElements"
-# - description: "CTAO Northern Array Element Coordinates"
-# - center_lon: "-17.8920302 deg"
-# - center_lat: "28.7621661 deg"
-# - center_alt: "2177m"
-# - center_northing: "3185066.278 m"
-# - center_easting: "217611.227 m"
-# - corsika_obs_level: "2158 m"
-# - EPSG: 32628
+# meta:
+#   EPSG: 32719
+#   center_alt: !astropy.units.Quantity
+#     unit: &id001 !astropy.units.Unit {unit: m}
+#     value: 2150.0
+#   center_easting: !astropy.units.Quantity
+#     unit: *id001
+#     value: 217654.06411838497
```

Comment:
Why did this value change?

Reply:
I got the values from the file but this was somehow mistaken. I now get the values from the `layout/telescope_positions-North-TestLayout.ecsv`. These values are still slightly different. It is a bit annoying that we have different sources of the same information that differ a bit. That is why I proposed in the email sent earlier today to create a versioning for these headers, so we know at which version the header and can update it with the latest version to be found in the DB

Reply:
Yes, having a version in the file might be useful. First I would like to get all the files to have the same info exactly and then perhaps we can add a comment with the version.